adhocracy4/projects/models.py

```diff
@@ -145,6 +145,18 @@ def other_projects(self):
     def is_private(self):
         return not self.is_public
 
+    @property
+    def modules(self):
+        return self.module_set
```

Comment:
If it is meant for external usage I would definitely return a queryset and not a related manager. In a addition one could also ask if we need the proposed abstraction right now.